README.md

```diff
@@ -1,2 +1,174 @@
 # qrcode.js
-A QrCode library
+![license](https://img.shields.io/github/license/mashape/apistatus.svg)
+![circleCi](https://circleci.com/gh/AdactiveSAS/qrcode.js.svg?style=shield&circle-token=:circle-token)
+
+<p align="center">
+  <a href="http://adactive.com">
+    <img alt="babel" src="https://user-images.githubusercontent.com/8574893/41667613-f9636b62-74df-11e8-9b9a-981f86a2a521.png">
+  </a>
+</p>
+
+## Introduction
+qrCode.js is a web based QrCode generation library in javascript. It extends the functionality of [qrcode-generator](https://github.com/kazuhikoarase/qrcode-generator). 
+
+## Getting started
+
+```javascript
+    npm i qrcode.es
+```
+OR
+```javascript
+    yarn add qrcode.es
+```
+
+## Usage
+
+### ES6 Modules
+```javascript
+import QrCode, {MODES} from 'qrcode.es';
```

Comment:
I don't think so 
 !

https://github.com/AdactiveSAS/qrcode.js/blob/f4baf58c05a5d24b9a182a2008a0a249c40a6286/src/index.js

You didn't test it ?!